connectors/sources/confluence.py

```diff
@@ -377,19 +473,47 @@ async def fetch_confluence_server_users(self):
             url = urljoin(self.host_url, URLS[USERS_FOR_SERVER])
 
         while True:
-            url_ = url.format(start=start_at, limit=limit)
-            users = await self.api_call(url=url_)
-            response = await users.json()
-            if len(response.get(key)) == 0:
-                return
-            yield response.get(key)
-            start_at += limit
+            try:
+                url_ = url.format(start=start_at, limit=limit)
+                users = await self.api_call(url=url_)
+                response = await users.json()
+                if len(response.get(key)) == 0:
+                    return
+                yield response.get(key)
+                start_at += limit
+            except (
+                ServerConnectionError,
+                ClientResponseError,
+                ClientPayloadError,
+                Forbidden,
+                UnauthorizedException,
+                ThrottledError,
+                NotFound,
+                InternalServerError,
+                Exception,
+            ) as exception:
```

Comment:
What's the purpose in having this whole list, but including `Exception`? Isn't this the same as
```
except Exception as exception:
```
?